libraries/chain/include/eosio/chain/log_catalog.hpp

```diff
@@ -1,17 +1,17 @@
 #pragma once
-#include <boost/container/flat_map.hpp>
-#include <boost/iostreams/device/mapped_file.hpp>
 #include <fc/io/cfile.hpp>
 #include <fc/io/datastream.hpp>
+#include <boost/filesystem/path.hpp>
```

Comment:
Why is boost/filesystem/path.hpp needed?

Reply:
Thanks. That looks like a bad merge. Replaced with `<filesystem>`.